local-proxy/src/server.ts

```diff
@@ -168,8 +171,15 @@ router.post('/invoke', async (ctx) => {
       }
       const mod = require(joinedDir);
       fn = mod[handler];
+      // Cast to any so typescript doesn't complain against accessing __shiftjs__ on a function
```

Comment:
Note that this has a conflict with https://github.com/reshufflehq/reshuffle/pull/219, which renames `__shiftjs__` to `__visibility__`.